exporter/signalfxexporter/README.md

```diff
@@ -86,6 +86,10 @@ The following configuration options can also be configured:
   defined in `translation/constants.go`.
 - `timeout` (default = 10s): Amount of time to wait for a send operation to
   complete.
+- `http2_read_idle_timeout` (default = 10s): If the connection has been idle for the configured value send a ping frame for health check.
+  0s means http/2 health check will be disabled.
+- `http2_ping_timeout` (default = 5s): Triggered by `http2_read_idle_timeout`; When there's no response to the ping within the configured value, 
+  the connection will be closed. If this value is set to 0, it will default to 15s. 
```

Comment:
Why would `0` be set to `15s`? Shouldn't it be set to the default value, in this case `5s`?

Reply:
@crobert-1 
This is the current behavior in the http2 library https://pkg.go.dev/golang.org/x/net/http2#Transport  and I am keeping it the same here, the confighttp is also behaving the same